app/containers/profilePanel.js

```diff
@@ -50,19 +56,19 @@ function ProfilePanel (props) {
     })
   }
 
-  function onClickBlockUserAll () {
-    props.moderationBlock({
-      addr: props.addr,
-      userKey: user.key
-    })
-  }
+  // function onClickBlockUserAll () {
+  //   props.moderationBlock({
+  //     addr: props.addr,
+  //     userKey: user.key
+  //   })
+  // }
 
-  function onClickUnblockUserAll () {
-    props.moderationUnblock({
-      addr: props.addr,
-      userKey: user.key
-    })
-  }
+  // function onClickUnblockUserAll () {
+  //   props.moderationUnblock({
+  //     addr: props.addr,
+  //     userKey: user.key
+  //   })
+  // }
```

Comment:
remove instead of comment out

(at least if it's block, which isn't implemented in master yet)